src/js/components/Navigation/SettingsPersonalSideBar.jsx

```diff
@@ -8,7 +8,7 @@ import { renderLog } from '../../common/utils/logging';
 import styled from 'styled-components';
 import DesignTokenColors from '../../common/components/Style/DesignTokenColors';
 import { ImportContactsOutlined, AccountBoxRounded, Lock, SecurityRounded, CampaignRounded, PeopleAltRounded, TextsmsRounded, ExitToAppRounded  } from '@mui/icons-material';
-
+import TagManager from 'react-gtm-module';
```

Comment:
Hi @rachnarawalpally if you run `npm run lint` from your terminal, I think it will complain about `import TagManager from 'react-gtm-module';` needing to be above `import styled...`.